service/lib/agama/cmdline_args.rb

```diff
@@ -35,6 +35,10 @@ def initialize(data = {})
       @data = data
     end
 
+    def self.read
+      read_from("/proc/cmdline")
```

Comment:
sadly this collides with https://github.com/agama-project/agama/pull/1896 In general agama should not read /proc/cmdline but generated one...and that brings me to topic maybe for retro, why we have after beta1 so many open pull requests?

Reply:
Having the `read` method is good, so you do not need to refer to `/proc/cmdline` everywhere. And we can adapt it whenever is needed.